activation/atxdb.go

```diff
@@ -524,7 +524,8 @@ func (db *DB) storeAtxUnlocked(atx *types.ActivationTx) error {
 		return err
 	}
 
-	atxBodyBytes, err := types.InterfaceToBytes(getAtxBody(atx))
+	// todo: this changed so that a full atx will be written - inherently there will be double data with atx header
+	atxBodyBytes, err := types.InterfaceToBytes(atx)
```

Comment:
so maybe let's rename as this is the full atx bytes and not just body..

Reply:
and we can remove the comment below

Reply:
why was it necessary to write the full ATX and not store the header separately?

Reply:
Since `ActivationTxHeader` is public it is written to DB anyways as zeros +
New sync paradigm is to store data with it's respected hash as key - so ATX needs to be full for validation